.github/workflows/build.yml

```diff
@@ -30,11 +31,17 @@ jobs:
         cargo fmt --manifest-path services/Cargo.toml --all -- --check
       if: matrix.rust == 'stable'
 
-    - name: sdk tests 
+    - name: build core for wasm
+      run: cargo build -p azure_core --target=wasm32-unknown-unknown
```

Comment:
build wasm

Reply:
Is building necessary? Can we just do `cargo check`?

Reply:
You are correct. `check` is sufficient.